content/docs/sidebar.json

```diff
@@ -416,6 +416,10 @@
         "label": "Get Started (GitHub)",
         "slug": "start-github"
       },
+      {
+        "label": "Get Started (GitLab)",
+        "slug": "get-started-with-cml-on-gitlab"
```

Comment:
This adds the nav entry on the left side of the website:

![image](https://user-images.githubusercontent.com/1477535/107544497-ea47c700-6b8f-11eb-8dca-6a67c51ae6b7.png)

Reply:
BTW the slug (will become the last part of the URL) should be `start-gitlab` I think, like in https://dvc.org/doc/cml/start-github. Specific change below:

```suggestion
        "label": "Get Started (GitHub)",
        "slug": "start-github"
      },
      {
        "label": "Get Started (GitLab)",
        "slug": "start-gitlab"
```

BUT this requires to also rename the content/docs/cml/get-started-with-cml-on-gitlab.md file to .../start-gitlab.md

Reply:
OK I see you added a file called start-gitlab.md so I'll merge this suggestion now. Since this will change online and not in your machine, please `git pull` @jendefig (or you won't be able to push).